tools/applicationinsights-web-snippet/rollup.config.js

```diff
@@ -46,6 +52,39 @@ const nodeUmdRollupConfigFactory = () => {
   return nodeRollupConfig;
 };
 
+const snippetNodeUmdRollupConfigFactory = () => {
+  const nodeRollupConfig = {
+    input: `${snippetInputName}.js`,
+    output: {
+      file: `${distPath}/${snippetOutputName}.js`,
+      banner: banner,
+      format: "umd",
```

Comment:
We don't want to generate a "snippet.min.js" using the "umd" module, we will probably want to use either "cjs" or "esm"...

cjs adds a "use strict" (which we don't really want)